10_rest/runnerdave/pkg/puppy/rest_test.go

```diff
@@ -0,0 +1,227 @@
+package puppy_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	chi "github.com/go-chi/chi"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	puppy "github.com/anz-bank/go-course/10_rest/runnerdave/pkg/puppy"
+	store "github.com/anz-bank/go-course/10_rest/runnerdave/pkg/puppy/store"
+)
+
+var (
+	rs puppy.RestStorer
+)
+
+func startAPIHandler() *chi.Mux {
```

Comment:
this doesn't really start anything, it's more like it wires it together.
I usually call this: setupRoutes or wireRoutes.

Reply:
fixed